src/history/index.js

```diff
@@ -14,10 +24,23 @@ export async function recordVersion(serviceId, documentType, content, snapshotId
     throw new Error(`A snapshot ID is required to ensure data consistency for ${serviceId}'s ${documentType}`);
   }
 
-  return record({
+  let id = '';
+  if (config.get('history.publish')) {
+    id += config.get('history.publicSnapshotsRepository')
+      .replace('git@github.com:', 'https://github.com/')
+      .replace('.git', '/commit/');
```

Comment:
This makes the assumption that the repository is published on GitHub (and is fragile as it expects the syntax to be SSH and end with `.git`). Let's avoid vendor lock-in and allow the prefix to be defined in config 🙂

Reply:
Ok but if remove the `replace`s I have to add a config key for the link in body commit message for the link to the snapshot commit. Are you ok with that? What about `snapshotsBaseUrl`?

Reply:
Yes, adding a config key was what I had in mind! `snapshotsBaseUrl` sounds good 👍